test/gtest/ucs/test_mpool.cc

```diff
@@ -73,6 +83,39 @@ class test_mpool : public ucs::test {
         return UCS_LOG_FUNC_RC_CONTINUE;
     }
 
+    static bool
+    check_chunk_elems_num(test_mpool_grow_factor_case_t &test_case)
+    {
+        unsigned chunk_num             = test_case.num_elems_per_chunk.size();
+        ucs_mpool_chunk_t *chunk       = test_case.mp.data->chunks;
+        const unsigned total_num_elems = test_case.mp.data->quota;
+        unsigned i;
+
+        for (i = 0; i < total_num_elems; ++i) {
+            if (ucs_mpool_get(&test_case.mp) == NULL) {
+                return false;
+            }
+        }
+
+        chunk = test_case.mp.data->chunks;
+        if (chunk == NULL) {
+            return false;
+        }
+
+        while (chunk != NULL && chunk_num > 0) {
```

Comment:
write it as for loop
add ( ) around sub-expressions

Reply:
fixed.